cypress/e2e/tests/pages/cluster-manager.spec.ts

```diff
@@ -0,0 +1,121 @@
+const { baseUrl } = Cypress.config();
```

Comment:
Once v2.6.7 is out we need to re-sync on PageObjects and how they can be used in this spec

Reply:
Yes, this is definitely something which we'll have to review having some practical cases and get some thought about it.
I would recommend to do that after https://github.com/rancher/dashboard/issues/5991

Reply:
I would require this to merged in such case though, otherwise I'll have to create another ticket with the configuration I'm using here and set merge it, then remove it from here 😬

Reply:
As I've created another issue, we can focus on the tests themselves for now.
https://github.com/rancher/dashboard/issues/6627

Reply:
Currently we're hard coding selectors directly in tests and i would see that as a blocker for this PR. 

The tests here are a practical case for POs, have a good set that others can use, and see the methodology of, is really important at this stage. That being said i've started creating them and just need to test the last two specs complete successfully. It would be nice to get them in to this PR.

Reply:
I've created https://github.com/cnotv/dashboard/pull/1 as a PR to your branch. I think we should either get that in here or soon after

Reply:
Since you did already a PR, could you please do it directly with https://github.com/rancher/dashboard/issues/6627 so you can take care of everything related to it?